scout/build/disease.py

```diff
@@ -24,6 +24,8 @@ def build_disease_term(disease_info: dict, alias_genes: dict = {}) -> dict:
     hgnc_symbols_not_found = set()
     hgnc_ids = set()
     for hgnc_symbol in disease_info.get("hgnc_symbols", []):
+        if hgnc_symbol is None:
```

Comment:
```suggestion
        if None in hgnc_symbol:
```

Reply:
That line should work, could you please check again? Before this change in the branch  I saw:

<img width="964" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/361087b9-d968-441f-8aef-784222bf9a81">

After the change:

<img width="820" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/2c9af073-c2b0-46e8-bae1-de9d22401399">